src/components/CheckEditor/FormComponents/MultiHttpCheckRequests.tsx

```diff
@@ -70,28 +73,26 @@ export const MultiHttpCheckRequests = () => {
   const requests = watch('settings.multihttp.entries');
 
   useEffect(() => {
-    const onErrorEvent = (e: CustomEvent<FieldErrors<CheckFormValues>>) => {
-      const errs = e.detail;
+    const openRequest = (e: CustomEvent<CheckFormInvalidSubmissionEvent>) => {
+      const { errs, source } = e.detail;
       const res = getMultiHttpFormErrors(errs);
 
-      if (res) {
+      if (res !== null) {
         dispatchCollapse({
-          type: 'updateRequestPanel',
-          open: true,
-          index: res.index,
-          tab: res.tab,
+          type: 'openRequestPanels',
+          indexes: res,
         });
 
-        if (panelRefs.current[res.index]) {
-          focusField(panelRefs.current[res.index], res.id);
+        if (source !== `collapsible`) {
```

Comment:
Can you expand a bit on this scenario? When does this execute when the `source` equals `collapsible`?

Reply:
I've added a comment explaining why it is needed to check the broadcast source (it's to avoid an infinite loop -- if you have two event listeners which rebroadcast the same event they continually trigger each other).